app/views/errors/forbidden.html.erb

```diff
@@ -1 +1,9 @@
-<h1>Forbidden</h1>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Page not found</h1>
+    <p class="govuk-body">If you typed the web address, check it is correct.</p>
+    <p class="govuk-body">If you pasted the web address, check you copied the entire address.</p>
+    <p class="govuk-body">If the web address is correct or you selected a link or button,
+      contact: <%= render MailToSupportComponent.new %></p>
```

Comment:
That's a funny message for a 403 page, but I guess we don't want do let people map our site by giving them information that a page is 'real but not allowed'.

Reply:
Alaric said that's a common pattern for Gov, that to a user perspective there's no difference between "you can't see this" and "this doesn't exist"

Reply:
Sure there is a difference, if it doesn't exist, you think that maybe you got a dead url. But if it exists and you can't see it, well, you might need to ask for some permissions. So for a determined user, different messages point them to different first course of action.

Whether having incorrect permissions is a common thing in government is a common problem or not, eh, who knows.